### ISSUE TITLE
Change addon name to be capitalized but not uppercase

### ISSUE THREAD

**Issue by andrei-ilyukovich** [tuchk4/storybook-readme#151]
It is just a suggestion to make addon name just like it is done for other storybook addons i.e. capitalized and not uppercase.

![image](https://user-images.githubusercontent.com/6919053/55717377-e41bb900-5a01-11e9-9bd6-fd87100be854.png)

**Comment by rjgux** [comment]
Or a config setting to override the label would be nice.

**Comment by tuchk4** [comment]
Thanks for lil feature request. Will be shipped with next release.

**Comment by tuchk4** [comment]
published 5.0.2

## Setup

Register addon at _.storybook/addons.js_

```js
import 'storybook-readme/register';
```

NOTE: It is possible to set addon's panel title

```js
import registerWithPanelTitle from 'storybook-readme/registerWithPanelTitle';

registerWithPanelTitle('Docs');
```
